packages/shopify-app-remix/src/server/authenticate/fulfillment-service/types.ts

```diff
@@ -97,21 +97,18 @@ export interface FulfillmentServiceContext<
    * import { authenticate } from "../shopify.server";
    *
    * export const action: ActionFunction = async ({ request }) => {
-   *   const { kind } = await authenticate.fulfillmentService(request);
-   *   if(kind === 'FULFILLMENT_REQUEST') {
+   *   const { payload } = await authenticate.fulfillmentService(request);
+   *   if(payload.kind === 'FULFILLMENT_REQUEST') {
    *    // handle fulfillment request
-   *   } else if (kind === 'CANCELLATION_REQUEST') {
+   *   } else if (payload.kind === 'CANCELLATION_REQUEST') {
    *    // handle cancellation request
    *   };
    * return new Response();
    * ```
    */
-  kind: FulfillmentServiceRequestKind;
-}
-
-export enum FulfillmentServiceRequestKind {
```

Comment:
Good call removing the enum. It's nice but it would tie this code to the requests, which could make it harder for apps to implement this.